moveit_ros/moveit_servo/src/parameter_descriptor_builder.cpp

```diff
@@ -0,0 +1,89 @@
+// Copyright 2021 PickNik Inc.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are met:
+//
+//    * Redistributions of source code must retain the above copyright
+//      notice, this list of conditions and the following disclaimer.
+//
+//    * Redistributions in binary form must reproduce the above copyright
+//      notice, this list of conditions and the following disclaimer in the
+//      documentation and/or other materials provided with the distribution.
+//
+//    * Neither the name of the PickNik Inc. nor the names of its
+//      contributors may be used to endorse or promote products derived from
+//      this software without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+// LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+// CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+// POSSIBILITY OF SUCH DAMAGE.
+
+#include "../include/moveit_servo/parameter_descriptor_builder.hpp"
+
+namespace parameter_set
+{
+ParameterDescriptorBuilder& ParameterDescriptorBuilder::type(uint8_t type)
+{
+  msg_.type = type;
+  return *this;
+}
+
+ParameterDescriptorBuilder& ParameterDescriptorBuilder::description(std::string description)
+{
+  msg_.description = description;
+  return *this;
+}
+
+ParameterDescriptorBuilder& ParameterDescriptorBuilder::additionalConstraints(std::string additional_constraints)
+{
+  msg_.additional_constraints = additional_constraints;
+  return *this;
+}
+
+ParameterDescriptorBuilder& ParameterDescriptorBuilder::readOnly(bool read_only)
+{
+  msg_.read_only = read_only;
+  return *this;
+}
+
+// In rolling (not in Foxy)
+// ParameterDescriptorBuilder& ParameterDescriptorBuilder::dynamic_typing(bool
+// dynamic_typing)
+// {
+//   msg_.dynamic_typing = dynamic_typing;
+//   return *this;
+// }
```

Comment:
You should be able to uncomment this.  Also, please look into the ParameterDescriptor to make sure there are not any new features that have been added since I did this.